### PR TITLE
Task.51992

### PR DESCRIPTION
**Pull request by dangthibichngocb**
## Related Tickets
- [#TicketID](https://edu-redmine.sun-asterisk.vn/issues/51992)

## WHAT (optional)
- set up theme admin page.

## HOW
- I edit js file, inject not_vary_normal items in calculate function.

## WHY (optional)
- Because in previous version - number just depends on `normal` items. But in new version, we have `state` and `confirm_state` depends on both `normal` + `not_normal` items.

### CODE REVIEW

File: app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -1,16 +1,36 @@
...
   <head>
-    <title>InternRuby2022FoodAndDrink</title>
-    <meta name="viewport" content="width=device-width,initial-scale=1">
+    <title><%= full_title yield :title %></title>
     <%= csrf_meta_tags %>
     <%= csp_meta_tag %>
 
-    <%= stylesheet_link_tag "application", media: "all", "data-turbolinks-track": "reload" %>
-    <%= javascript_pack_tag "application", "data-turbolinks-track": "reload" %>
+    <%= stylesheet_link_tag 'application', media: 'all', 'data-turbolinks-track': 'reload' %>
```
Comment (thonght-3051) [inline app/views/layouts/application.html.erb]: dùng dấu " đi em

File: app/views/layouts/application.html.erb [hunk 2]
```diff
@@ -1,16 +1,36 @@
...
+    <%= javascript_pack_tag 'application', 'data-turbolinks-track': 'reload' %>
   </head>
 
-  <body>
-    <%= yield %>
+  <body class="hold-transition skin-blue sidebar-mini">
+      <div class="wrapper">
```
Comment (thonght-3051) [inline app/views/layouts/application.html.erb:12]: convention nè em

File: babel.config.js [hunk 3]
```diff
@@ -1,78 +1,78 @@
 module.exports = function(api) {
-  var validEnv = ["development", "test", "production"]
+  var validEnv = ['development', 'test', 'production']
   var currentEnv = api.env()
-  var isDevelopmentEnv = api.env("development")
-  var isProductionEnv = api.env("production")
-  var isTestEnv = api.env("test")
+  var isDevelopmentEnv = api.env('development')
+  var isProductionEnv = api.env('production')
+  var isTestEnv = api.env('test')
```
Comment (thonght-3051) [inline babel.config.js]: dùng dấu " em

File: config/database.yml [hunk 4]
```diff
@@ -0,0 +1,55 @@
+# MySQL. Versions 5.5.8 and up are supported.
```
Comment (thonght-3051) [inline config/database.yml]: file này ignore đi em, tạo 1 cái file example í

File: config/locales/vi.yml [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+    nav:
+      category: "Loại"
+      product: "Đồ uống và Đồ ăn"
+    header:
+      home: "Trang chủ"
+      contact: "Liên hệ"
+      call_me: "tất cả tôi bất cứ khi nào bạn có thể ..."
+      hour: "giờ"
+      msg: "tin nhắn"
+      min: "phút"
```
Comment (thonght-3051) [inline config/locales/vi.yml]: end line

File: config/webpacker.yml [hunk 6]
```diff
@@ -7,9 +7,10 @@ default: &default
   public_output_path: packs
   cache_path: tmp/cache/webpacker
   webpack_compile_output: true
+  resolved_paths: ['app/assets']
```
Comment (thonght-3051) [inline config/webpacker.yml]: dấu "

**Review by thonght-3051 — APPROVED** [review]

**Comment by dangthibichngocb** [comment]
ready

**Pull request merged**